[docs] Release notes for 2.20.2.0 (#21262)
yugabyte/yugabyte-db: docs/content/preview/releases/yba-releases/v2.20.md

{{< /warning >}}

## v2.20.2.0 - March 4, 2024 {#v2.20.2.0}

**Build:** `2.20.2.0-b145`

**Third-party licenses:** [YugabyteDB](https://downloads.yugabyte.com/releases/2.20.2.0/yugabytedb-2.20.2.0-b145-third-party-licenses.html), [YugabyteDB Anywhere](https://downloads.yugabyte.com/releases/2.20.2.0/yugabytedb-anywhere-2.20.2.0-b145-third-party-licenses.html)

### Download

<ul class="nav yb-pills">
 <li>
   <a href="https://downloads.yugabyte.com/releases/2.20.2.0/yba_installer_full-2.20.2.0-b145-linux-x86_64.tar.gz">
     <i class="fa-brands fa-linux"></i>
     <span>Linux x86</span>
   </a>
 </li>
</ul>

### New features

<!-- * Exposes the `replicated-migrate` flag as an {{<badge/ea>}} feature with modified help text, making commands accessible when `YBA_MODE` is set to `EA`, improving the user interface of scheduled backups along with button visibility. PLAT-11274,PLAT-11096 -->

* [Nutanix Ready](../../../integrations/nutanix-ahv/). YugabyteDB and YugabyteDB Anywhere has been certified to be compatible as part of the Nutanix Ready AHV program.

> * Automated [Linux OS patching](../../../yugabyte-platform/manage-deployments/upgrade-nodes/) and upgrades for on-prem. You can now utilize safety-improved YugabyteDB  Anywhere APIs when automating you rolling patch scripts for Linux OSs (typically, for OS security, and minor or major OS upgrades). PLAT-10591, PLAT-10988

Reviewer: ```suggestion
* Automated [Linux OS patching](../../../yugabyte-platform/manage-deployments/upgrade-nodes/) and upgrades for on-prem. You can now use safety-improved YugabyteDB  Anywhere APIs when automating your rolling patch scripts for Linux OSs (typically, for OS security, and minor or major OS upgrades). PLAT-10591, PLAT-10988
```

* REST API versioning and clean-up. REST API simplification and streamlining for most important APIs. PLAT-11133

> * YugabyteDB Anywhere (YBA) and YugabyteDB version compatibility list. You can now independently upgrade YBA without upgrading the YugabyteDB universes. All supported YugabyteDB versions are compatible. PLAT-11260

Reviewer: True?
Ticket number doesn't look related

* Improved Auth when using [GCS for backups for Kubernetes universes](../../../yugabyte-platform/back-up-restore-universes/configure-backup-storage/#google-cloud-storage). When taking backups of a Kubernetes universe to Google GCP, allow use of the Google GKE's Service Account for authentication to the Google Storage. PLAT-10571

* Asymmetric nodes in each region (API support only). In a stretched cluster, nodes in each region can now have different instance types (that is, different number of CPU cores, memory, and so on).

> * [Task Retryability](/stable/yugabyte-platform/manage-deployments/retry-failed-task/) for VM-based universes. You can now address the cause of a failure for a complex operation fails and retry. For example, if adding nodes to a universe fails ( due to a cloud quota limit), you can now address the cloud issue and retry. {{<badge/ea>}} PLAT-6141

Reviewer: ```suggestion
* [Task Retryability](/stable/yugabyte-platform/manage-deployments/retry-failed-task/) for VM-based universes. You can now address the cause of a failure for a complex operation and retry. For example, if adding nodes to a universe fails due to a cloud quota limit, you can fix the quota issue and retry the task. {{<badge/ea>}} PLAT-6141
```

> * [Backup portability](../../../yugabyte-platform/back-up-restore-universes/restore-universe-data/#advanced-restore-procedure). After sending backups to a cloud object store (For example, S3, GCS, Azure blob storage), you can now copy or transfer those backup sets to other cloud storage buckets and still restore successfully. {{<badge/ea>}} PLAT-10179

Reviewer: ```suggestion
* [Backup portability](../../../yugabyte-platform/back-up-restore-universes/restore-universe-data/#advanced-restore-procedure). After sending backups to a cloud object store (such as S3, GCS, or Azure blob storage), you can now copy those backup sets to other cloud storage buckets and restore from them. {{<badge/ea>}} PLAT-10179
```

> * [xCluster DR](../../../yugabyte-platform/back-up-restore-universes/disaster-recovery/). A turnkey solution for Business Continuity and Disaster Recovery to recover from an unplanned outage and planned maintenance. Replication occurs asynchronously (using [xCluster Replication](../../../architecture/docdb-replication/async-replication/)) from one universe to another. {{<badge/ea>}}

Reviewer: ```suggestion
* [xCluster DR](../../../yugabyte-platform/back-up-restore-universes/disaster-recovery/). A turnkey solution for business continuity and disaster recovery, allowing you to recover from an unplanned outage (failover) or to perform a planned switchover. {{<badge/ea>}}
```

> * [Edit universe's provider](../../../yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/gcp/): add/remove regions, change service account. Expanded support for reconfiguring a universe. You can now add or remove an availability zone/region/or an AMI, and modify CSP credentials when a universe's provider is actively in-use.

Reviewer: ```suggestion
* [Edit universe providers](../../../yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/gcp/). You can now add and remove regions and change service accounts for providers with active universes.
```
